Assignment3/Assignment3_QNo1/SolAssignmentNo3_QNo1.rb

```diff
@@ -0,0 +1,29 @@
+# => 1. Write a program to get all the words which are starting from 'ab'.
+# => arr = ["abcde", "abdf", "adeab", "abdgse", "bdefa", "bacdef"]
+# => Main buisness logic
+def get_initial_array
+  # => Enter the size of array
+  puts "Enter the size of array"
+  size = gets.chomp.to_i
+  # => check for validity
+  if size <= 0
+    puts "Sorry size must be positive"
+    Kernel.exit(0)
+  end
+  # => Allocate to size of array
+  arr = Array.new(size)
+  # => Enter each element
+  (1..size).each  { |cnt|
+    puts "Enter the #{cnt} element"
+    arr << gets.chomp
+  }
```

Comment:
@HrisikeshSinghaChowdhury Please use `do ... end` instead of using `{ ... }` for multi line block.